### PR TITLE
[MT] Add completion rate

### PR DESCRIPTION
**Pull request by glictevo**

### CODE REVIEW

**Review by Anders-E — APPROVED** [review]

File: rest_api/v2/paths/business/post_update.yml [hunk 1]
```diff
@@ -64,6 +64,8 @@ requestBody:
             $ref: '../../components/index.yml#/schemas/Photos'
           national_identification_number:
             $ref: '../../components/index.yml#/schemas/BusinessNationalIdentificationNumber'
+          completion_rate:
```
Comment (psykeron) [inline rest_api/v2/paths/business/post_update.yml:67]: We should not document this for the request. The document is public whereas the team's implementation of the completion rate is meant for private/internal use.

right?

File: rest_api/v2/paths/business/post_create.yml [hunk 2]
```diff
@@ -64,6 +64,8 @@ requestBody:
             $ref: '../../components/index.yml#/schemas/Photos'
           national_identification_number:
             $ref: '../../components/index.yml#/schemas/BusinessNationalIdentificationNumber'
+          completion_rate:
+            $ref: '../../components/index.yml#/schemas/BusinessCompletionRate'
```
Comment (psykeron) [inline rest_api/v2/paths/business/post_create.yml:68]: We should not document this for the request. The document is public whereas the team's implementation of the completion rate is meant for private/internal use.

right?

**Review by psykeron — CHANGES_REQUESTED** [review]
need to address comments

File: rest_api/v2/paths/business/post_create.yml (hunk 2 above)
Comment (Anivial) [reply rest_api/v2/paths/business/post_create.yml:68]: btw it should not be in the business POST, but business search/get only.
i'll fix it

File: rest_api/v2/paths/business/post_update.yml (hunk 1 above)
Comment (Anivial) [reply rest_api/v2/paths/business/post_update.yml:67]: It should be only available in business get/search, and not in business create/update, you are right.
I'll fix this

**Review by psykeron — APPROVED** [review]
🎸

**Pull request merged**